changelogs/fragments/1481-deprecated-brew-cask-command.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - homebrew_cask - Homebrew will be deprecating use of ``brew cask`` commands as of version 2.6.0, see https://brew.sh/2020/12/01/homebrew-2.6.0/. Added logic to stop using ``brew cask`` for brew version >= 2.6.0 (https://github.com/ansible-collections/community.general/pull/1481)
```

Comment:
```suggestion
  - homebrew_cask - Homebrew will be deprecating use of ``brew cask`` commands as of version 2.6.0, see https://brew.sh/2020/12/01/homebrew-2.6.0/. Added logic to stop using ``brew cask`` for brew version >= 2.6.0 (https://github.com/ansible-collections/community.general/pull/1481).
```
**Please** stop just applying *parts* of the suggestion. This is **really** annoying and tiresome.

Reply:
oops my bad.